blog/draft/hoe-haven-jou-helpt.md

```diff
@@ -0,0 +1,117 @@
+
+**Basisdiensten:**
+
+- Computing: AWS EC2, Azure VMs, Google Compute Engine
+- Object storage: AWS S3, Azure Blob Storage, Google Cloud Storage
+- Databases: AWS RDS/DynamoDB, Azure SQL/Cosmos DB, Google Cloud SQL
+
+**Groeidiensten:**
+
+- Serverless: AWS Lambda, Azure Functions, Google Cloud Functions
+- Containers: AWS EKS/ECS, Azure Kubernetes Service, Google GKE
+- AI/ML: AWS SageMaker, Azure ML, Google Vertex AI
+- Data analytics: AWS Redshift, Azure Synapse, Google BigQuery
+
+![Haven](./img/stadskantoor_4.jpg)
+*Stadskantoor in Utrecht. Foto door [Eric de Redelijkheid](https://www.flickr.com/photos/ericdere/36957944765/). [Licentie: Deed - Attribution 2.0 Generic - Creative Commons ](https://creativecommons.org/licenses/by/2.0/deed.en)*
+
+## Azure/ AWS werden de defacto standaard
+Door het brede scala aan services en de schaalbaarheid werden Azure en AWS extreem populaire cloudproviders. Doordat deze bedrijven vroeg zijn begonnen met het investeren in deze sector en er in Amerika makkelijk toegang was tot tech-talent hadden ze grote voordelen. Ook hadden deze bedrijven het voordeel dat kapitaal des tijds goedkoop voorhanden was en maakten ze slim gebruik van ecosystemen van partner-bedrijven. 
+
+## Strategisch slim?
+Doordat Microsoft al jaren dé grote it-leverancier is voor de Nederlandse overheid, was de overstap op Microsoft Azure een logische. Echter rijst nu de vraag: is het handig om strategisch zo afhankelijk te zijn van een Amerikaans bedrijf? Het is bekend dat de Amerikaanse overheid mee mag kijken in de infrastructuur van Amerikaanse bedrijven als zij dat wil, ook als de servers daarvan op Europees grondgebied staan [^4]. Servers op Europees grondgebied zijn dus ook geen oplossing voor dit probleem, zolang het Amerikaanse bedrijven betreft.
+
+## De Haven-checks als handvatten
+Dan naar de oplossing. Als je als organisatie toe beweegt naar een situatie waarin je er voor zorgt dat je Kubernetes-clusters allemaal Haven-compliant zijn, werk je toe naar een gestandaardiseerde situatie. Dit zorgt er voor dat het een stuk **makkelijker wordt** om vervolgens later de boel te migreren naar een **Europese cloudprovider**.
+
+## Haven+ als vervanging voor Azure/AWS services
+Je hebt inderdaad niet direct vervanging voor je mooie glimmende Azure/AWS services. Maar je kan ze wel langzamerhand gaan vervangen door open source componenten die je zelf opzet. En het mooie is; een aantal van deze open source componenten heeft het Haven-team al voor je beschikbaar gemaakt in de vorm van een project genaamd [**Haven+**](https://gitlab.com/commonground/haven/havenplus).
```

Comment:
Willen we niet linken naar onze eigen kennisbank?

Reply:
Yes, pas ik aan